File: lecture_001/greatest.java
import java.util.Scanner;
public class greatest{
    public static Scanner scn=new Scanner(System.in);
    public static void main (String[] args){
        System.out.println("enter three numbers to be varifyed");
        int a=scn.nextInt();
        int b=scn.nextInt();
        int c=scn.nextInt();
        System.out.println("the largest number is");
        if (a>b && a>c)
        System.out.println(a);
        else if (b>c && b>a)
        System.out.println(b);
        else
        System.out.println(c);
    }
}

File: lecture_001/greatest.cpp
#include<iostream>
using namespace std;
int main(int args, char**argv){
    cout<<"enter numbers you want to verify"<<endl;
    int a,b,c;
    cin>>a>>b>>c;
    cout<<"the greatest number is \n";
    if(a>b && a>c)
    {cout<<a;}
    else if (b>c && b>a)
    cout<<b;
    else if (c>b && c>a)
    cout<<c;
    return 0;
}